resources/views/races/members/_form.blade.php

```diff
@@ -0,0 +1,4 @@
+<div class="form-group">
+    <label>{{ __('races.fields.members') }}</label>
+    <select multiple="multiple" name="members[]" id="members" class="form-control form-members" style="width: 100%" data-url="{{ route('characters.find', ['with_race' => '1']) }}"></select>
```

Comment:
What's `with_race`? I don't see anything in the code else regarding this property. Is it needed?